File: public/examples/integrations/toolkits/search/google_shopping/search_shopping_products_example_call_tool.js
import { Arcade } from "@arcadeai/arcadejs";

const client = new Arcade(); // Automatically finds the `ARCADE_API_KEY` env variable

const TOOL_NAME = "GoogleShopping.SearchProducts";

const toolInput = {
	keywords: "Apple iPhone",
	language_code: "en",
	country_code: "us",
};

const response = await client.tools.execute({
	tool_name: TOOL_NAME,
	input: toolInput,
});

console.log(response);

File: public/examples/integrations/mcp-servers/search/google_shopping/search_shopping_products_example_call_tool.py
from arcadepy import Arcade

client = Arcade()  # Automatically finds the `ARCADE_API_KEY` env variable

TOOL_NAME = "GoogleShopping.SearchProducts"

tool_input = {
    "keywords": "Apple iPhone",
    "language_code": "en",
    "country_code": "us",
}

response = client.tools.execute(
    tool_name=TOOL_NAME,
    input=tool_input,
)
print(response)
